tests/stub/driver_parameters/test_bookmark_manager.py

```diff
@@ -474,16 +474,182 @@ def test_should_resolve_database_name_with_system_bookmarks(self):
     def _new_driver(self, bookmark_manager_config=None):
         if bookmark_manager_config is None:
             bookmark_manager_config = Neo4jBookmarkManagerConfig()
-
         uri = "neo4j://%s" % self._router.address
         auth = types.AuthorizationToken("basic", principal="neo4j",
                                         credentials="pass")
         return Driver(
             self._backend,
             uri, auth,
-            bookmark_manager_config=bookmark_manager_config
+            bookmark_manager_config=bookmark_manager_config)
+
+    def test_should_call_bookmark_supplier_for_all_get_bookmarks_call(self):
+        self._start_server(self._router, "router_with_db_name.script")
+        self._start_server(self._server, "transaction_chaining.script")
+
+        adb_bookmarks = ["adb:bm1"]
+        get_bookmarks_calls = []
+
+        def get_bookmarks(db):
+            get_bookmarks_calls.append([db])
```

Comment:
Why do you wrap it in a list? It's just a single string :thinking:

Reply:
Functions receives a list of params, it just for making easier to realise afterwards since most of mock frameworks returns the func calls as list of lists.

Reply:
> most of mock frameworks returns the func calls as list of lists

That's not the case in Python since positional and keyword-arguments are not strictly separated by default. But I don't mind it as it is right now.